//公会解散
message DseGuildDismiss
{
	optional int32 res = 1;
}